Tests.Integration.Presentation.Web/SystemUsage/V2/ItSystemUsageApiV2Test.cs

```diff
@@ -1835,6 +1880,26 @@ private static void AssertGDPR(GDPRWriteRequestDTO gdprInput, GDPRRegistrationsR
             Assert.Equal(gdprInput.DataRetentionEvaluationFrequencyInMonths ?? 0, gdprResponse.DataRetentionEvaluationFrequencyInMonths);
         }
 
+        private GeneralDataUpdateRequestDTO CreateValidityInput()
+        {
+            return new GeneralDataUpdateRequestDTO
+            {
+                Validity = new ValidityWriteRequestDTO
+                {
+                    ValidFrom = DateTime.UtcNow.AddDays(-1).Date,
+                    ValidTo = DateTime.UtcNow.AddDays(1).Date,
+                    LifeCycleStatus = A<LifeCycleStatusChoice>()
+                }
+            };
+        }
+
+        private static void AssertValidity(GeneralDataUpdateRequestDTO validityInput, GeneralDataResponseDTO validityResponse)
+        {
+            Assert.Equal(validityInput.Validity?.ValidFrom, validityResponse.Validity?.ValidFrom);
+            Assert.Equal(validityInput.Validity?.ValidTo, validityResponse.Validity?.ValidTo);
+            Assert.Equal(validityInput.Validity?.LifeCycleStatus, validityResponse.Validity?.LifeCycleStatus);
```

Comment:
I'm not sure if that's the correct way to test it. The "Validity" property is not being mapped in the ItSystemUsage[...]Mapper, only it's properties are being mapped, so the "Validity" itself won't be null 
![image](https://user-images.githubusercontent.com/101557040/191004563-9aeefcee-fda9-4dc7-9aa2-edeac9b0647c.png)